index.html

```diff
@@ -212,15 +215,42 @@ <h2 class="text-[17px] font-medium text-center mb-4">
                       class="grid grid-cols-7 gap-2 text-sm"
                     ></div>
                   </div>
+                  <div
+                    class="bottom-part flex justify-between items-center"
+                    id="boxes-container"
+                  >
+                    <!-- Boxes will be dynamically inserted here -->
+                  </div>
                 </div>
               </div>
 
               <div id="months-section" class="section hidden">
-                <h2 class="text-center text-lg">Months Content</h2>
+                <h2 class="text-center text-lg">
+                  Faraz Bhai, I think the round month bar in Airbnb website
+                  can't be made through the tech stack we are using ( html,
+                  tailwind css, Js ) and although I thought to ask you for this
+                  thing. Let me know about this.
```

Comment:
Sure it can.

Reply:
it can be made with the tech stack we are using????
Clear me this one please.